sim_pipeline/Pipelines/skypy_pipeline.py

```diff
@@ -28,7 +28,7 @@ def __init__(self, skypy_config=None, sky_area=None, filters=None):
 
             if sky_area is not None:
                 old_fsky = "fsky: 0.1 deg2"
-                new_fsky = f"fsky: %s %s" % (sky_area.value, sky_area.unit)
+                new_fsky = f"fsky: {sky_area.value} {sky_area.unit}"
```

Comment:
is there a particular reason for that change? Just curious as it worked for me in the old version

Reply:
Yes, with the old version, we get pre-commit warning: f-string without any placeholders.